content/250-postgres/1200-more/1000-faq.mdx

```diff
@@ -73,6 +75,143 @@ Yes, read and write queries are counted equally as _operations_ and are billed t
+
+- **Use nested-relation helpers** such as [`connectOrCreate`](/orm/reference/prisma-client-reference#connectorcreate) or [`set`](/orm/reference/prisma-client-reference#set) to create or link related records in a single operation.
+
+  ```ts
+  // Post and (if needed) its author, all in one request
+  await prisma.post.create({
+    data: {
+      title: 'Hello World',
+      author: {
+        connectOrCreate: {
+          where:  { email: 'alice@example.com' },
+          create: { name: 'Alice', email: 'alice@example.com' },
+        },
+      },
+    },
+  })
+  ```
+
+- **Prefer regular (array) [transactions](/orm/prisma-client/queries/transactions#transaction-api) over [interactive transactions](/orm/prisma-client/queries/transactions#interactive-transactions)** when the individual queries don't depend on each other.
+
+  ```ts
+  // Interactive transaction: counted as 2 operations
+  await prisma.$transaction(async (tx) => {
+    await tx.user.create({ data: { name: 'Alice' } })
+    await tx.post.create({ data: { title: 'Hello', authorId: 1 } })
+  })
+
+  // Array transaction: counted as 1 operation
+  await prisma.$transaction([
+    prisma.user.create({ data: { name: 'Alice' } }),
+    prisma.post.create({ data: { title: 'Hello', authorId: 1 } }),
+  ])
+  ```
+
+If a later query needs the result of an earlier one (for example, you need the user ID you just created), stick with an interactive transaction for correctness. Otherwise, batching and array transactions let you collapse multiple queries into a single billed operation, keeping both your operation count, and your cost down.
+
+### Is there a sample workload to estimate my expected charges?
+
+We will demonstrate three example workloads and estimate bills for small, medium, and large workloads. Each combine a realistic number of monthly active users (MAUs), a typical level of daily activity per user, and a rounded storage footprint. Here are the assumptions about the workloads:
+
+- **Small**: A hobby or early-stage side-project with ~`500` MAUs. Each user clicks around `10` times a day, and the entire database uses about `0.5` GB of storage.
+- **Medium**: A growing SaaS product serving ~`5000` MAUs. Power users average `40` actions per day, and the app stores roughly `6` GB of data.
+- **Large**: A production-grade, consumer-facing application handling ~`50000` MAUs. Heavy usage with `60` actions per user per day drives significant traffic, and the dataset reaches ~`40` GB.
+
+
+:::note[Plan rates]
+
+Here are the rates for each pricing plan:
+
+- **Starter plan** - \$18 per million operations
+  - Storage - 1 GB free then \$2 per additional GB
+- **Pro plan** - \$8 per million operations
+  - Storage - 5 GB free then \$1.5 per additional GB
+- **Business Annual plan** - \$4 per million operations
+  - Storage - 10 GB free then \$1 per additional GB
+
+You can also learn more about the rates for each plan on the [pricing page](https://www.prisma.io/pricing?utm_source=docs).
```

Comment:
We don't use the word "rates" generally. Could we use a terminology that is more aligned with what we usually use? E.g. pricing details?